api/lanes.py

```diff
@@ -1139,7 +1141,13 @@ def default_facet(cls, config, facet_group_name):
         return cls.SETTINGS[facet_group_name]
 
 
-class CrawlableCollectionBasedLane(DynamicLane):
+class CrawlableLane(DynamicLane):
+
+    # Crawlable feeds are cached for 12 hours.
+    MAX_CACHE_AGE = 12 * 60 * 60
```

Comment:
This should be fine in most cases, but it's too long for a shared ODL collection feed so maybe CrawlableCollectionBasedLane needs to override this.